src/Types/global.d.ts

```diff
@@ -1,26 +1,17 @@
-
 declare global {
-    type Displayable = string | JSX.Element
-    type Dict<K extends string | number, V> = {
-        [key in K]?: V
-    }
-    type StrictDict<K extends string | number, V> = {
-        [key in K]: V
-    }
+  type Displayable = string | JSX.Element
+  type Dict<K extends string | number, V> = Partial<Record<K, V>>
+  type StrictDict<K extends string | number, V> = Record<K, V>
```

Comment:
why are we keeping `StrictDict` if `Record` is 4 chars shorter?

Reply:
Let's keep them for now. Renaming `StrictDict` covers a lot more ground w.r.t. this PR, which mostly stays within `Formula`.

Reply:
If you want to rip the bandage, let's pick the pair we'll use moving forward.